File: JUNGOL/Beginner/#5934 별삼각형2/별삼각형2.cpp
#include <iostream>

using namespace std;

int main() {
	int n; cin>>n;

	if(n<1 || n>100 || !(n%2)) cout<<"INPUT ERROR!\n";
	else{
		int a = n/2+1;
		for(int i=a; i>=1; i--){
			for(int j=0; j<a-i; j++) printf(" ");
			for(int j=0; j<i; j++) printf("*");
			cout<<endl;
		}

		int b = a-1;
		for(int i=2; i<=a; i++){
			for(int j=0; j<b; j++) printf(" ");
			for(int j=0; j<i; j++) printf("*");
			cout<<endl;
		}
	}
}

File: JUNGOL/Beginner/#5934 별삼각형2/별삼각형2.py
def main():
    n = int(input())

    if n<1 or n>100 or n%2==0: print("INPUT ERROR!")
    else:
        a = n//2+1
        for i in range(a, 0, -1):
            print(' '*(a-i), '*'*i, sep='')
        
        b = a-1
        for i in range(2, a+1):
            print(' '*b, '*'*i, sep='')

main()
